hca/util/__init__.py

```diff
@@ -274,8 +274,18 @@ def __init__(self, config=None, swagger_url=None, **session_kwargs):
 
     @staticmethod
     def _audience(self):
-        return _deep_get(self.swagger_spec,["securityDefinitions","OauthSecurity","x-audience"]) or \
-               "https://auth.ucsc-cgp-redwood.org"
+        return _deep_get(self.swagger_spec, ["securityDefinitions", "OauthSecurity", "x-audience"]) or \
+            "https://auth.ucsc-cgp-redwood.org"
```

Comment:
May want to grab `os.environ['OIDC_AUDIENCE']` instead of hard-coding this

Reply:
that `OIDC_AUDIENCE` env var is for the `data-store` repo, im not sure if we would want to have that external dependency here....... :gir

Reply:
We can address this at a later time, merge away!